releasenotes/notes/add-fidelity-quantum-kernel-d40278abb49e19b5.yaml

```diff
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    Added a new implementation of quantum kernel. This implementation leverages the fidelity
```

Comment:
Maybe we can more say something like... this introduces Quantum Kernels based on (runtime) primitives that will allow better performance, and use of noise mitigation etc provided by the primitives. Something with a bit more of a top level statement rather than the matter of fact "new implementation"

Reply:
Slightly updated. I don't want to write things like "better performance, and use of noise mitigation" as it's unrelated to QML.

Reply:
In someways though this is a big piece the rationale around going over to primitives. I imagine there will be some sort of migration section to cover going from the old to the new. That can cover a more general statement I imagine.

Reply:
Updated the reno. I'm not sure if I addressed all you comments there.